src/wasm-s-parser.h

```diff
@@ -450,8 +450,18 @@ class SExpressionWasmBuilder {
             if (op[2] == 'p') return makeBinary(s, BinaryOp::CopySign, type);
             if (op[2] == 'n') {
               if (op[3] == 'v') {
-                if (op[8] == 's') return makeUnary(s, op[11] == '3' ? UnaryOp::ConvertSInt32 : UnaryOp::ConvertSInt64, type);
-                if (op[8] == 'u') return makeUnary(s, op[11] == '3' ? UnaryOp::ConvertUInt32 : UnaryOp::ConvertUInt64, type);
+                if (op[8] == 's') {
```

Comment:
Yes, it was not parsing some operators. Some issue for some other operators too. Just too much work here. I don't even like this style of parsing, it's ignoring much information and failing to catch errors. Would have liked to have had a symbol table and read and lookup each operator in the table, including the type prefix which seem unnecessary to split out and overly complicates many paths.